python/testSrc/com/jetbrains/python/PyTypeTest.java

```diff
@@ -816,6 +816,12 @@ public void testDefaultTupleParameterMember() {
            "  expr, foo = xs\n");
   }
 
+  // PY-19826
+  public void testListFromTupleType() {
+    doTest("List[Union[str, int]]",
+           "t = ('1',2,3); expr = list(t)");
+  }
+
```

Comment:
Does this test still pass even after changing the implementation of getElementTypes to return type of i-th element in tuple?